LoRaEngine/LoraKeysManagerFacade/FunctionsBundler/DeduplicationExecutionItem.cs

```diff
@@ -69,9 +81,48 @@ internal async Task<DuplicateMsgResult> GetDuplicateMessageResultAsync(DevEui de
 
                         if (updateCacheState)
                         {
+                            var previousGateway = cachedDeviceState.GatewayId;
+
                             cachedDeviceState.FCntUp = clientFCntUp;
                             cachedDeviceState.GatewayId = gatewayId;
                             _ = deviceCache.StoreInfo(cachedDeviceState);
+
+                            logger?.LogDebug("Previous connection owner was '{PreviousConnectionOwner}', current message was received from '{Gateway}'", previousGateway, gatewayId);
```

Comment:
Also here I'm not sure we want to log this. This means every function invocation to the bundler will now log this, when in debug mode. I would suggest to limit the logging to:
1. connection ownership assignment (first time -> no entry, or no gateway id)
2. connection ownership changes

Reply:
done in d489a749d8a517f62eb764a7522091ec36527b1c